### DISCUSSION TITLE
Upstream service responded with an unexpected status.

### DISCUSSION THREAD

**Discussion by andreha2023** [tillsteinbach/WeConnect-mqtt#250] [Q&A]
Hello,
Since yesterday I can no longer read wpassat/vehicles/*/domains/fuelStatus/rangeStatus/primaryEngine/remainingRange_km correctly. I get the following error message in MQTT Explorer: Upstream service responded with an unexpected status. If the problem persists, please contact our support.

Does anyone else have the problem? Maybe it's only temporary. I can't say right now whether the other data on the car is up to date, but I don't think so.

Many thanks for the help!

Best regards
Andre

**Comment by tillsteinbach** [comment]
For my passat this is also regularly a temporary problem. You also see it in the Volkswagen App when the range is gone. I believe that the data is removed when the backend is not sure if the value is still correct because the car was off-line for some time

**Comment by andreha2023** [comment]
hey,
unfortunately doesn't seem to be a temporary problem. The data in the Weconnect app is up to date. I also get the data for the mileage, but unfortunately not the important data, such as the remaining range of the electric drive. Very strange.

**Reply by tillsteinbach** [reply]
That is indeed very strange. In particular that it is the gasoline engine not reporting range. I only know that from the battery.
